essentials-module-teleport/src/main/kotlin/io/github/nyliummc/essentials/commands/TPACommand.kt

```diff
@@ -0,0 +1,179 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) NyliumMC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.github.nyliummc.essentials.commands
+
+import com.mojang.brigadier.CommandDispatcher
+import com.mojang.brigadier.context.CommandContext
+import com.mojang.brigadier.suggestion.Suggestions
+import com.mojang.brigadier.suggestion.SuggestionsBuilder
+import io.github.nyliummc.essentials.api.EssentialsMod
+import io.github.nyliummc.essentials.api.builders.Command
+import io.github.nyliummc.essentials.api.builders.TeleportRequest
+import io.github.nyliummc.essentials.api.builders.Text
+import io.github.nyliummc.essentials.configs.TeleportConfig
+import io.github.nyliummc.essentials.entities.TPACache
+import net.minecraft.command.arguments.EntityArgumentType
+import net.minecraft.server.command.ServerCommandSource
+import net.minecraft.text.LiteralText
+import net.minecraft.util.Formatting
+import java.util.concurrent.CompletableFuture
+
+object TPACommand {
+    val config by lazy {
+        EssentialsMod.instance.registry.getConfig(TeleportConfig::class.java)
+    }
+
+    fun register(dispatcher: CommandDispatcher<ServerCommandSource>) {
+        Command.builder(dispatcher) {
+            command("tpa") {
+                argument("user", EntityArgumentType.player()) {
+                    executes(::executeTpa)
+                }
+            }
+
+            command("tpahere") {
+                argument("user", EntityArgumentType.player()) {
+                    executes(::executeTpahere)
+                }
+            }
+
+            command("tpaccept") {
```

Comment:
wouldn't it make more sence to have tpaaccept? As it's for tpa and not tp

Reply:
This is how it's named in all other mods and plugins, we're keeping it like this for consistency.